### PR TITLE
Add support for removing a product from a syncplan

### PR DESCRIPTION
**Pull request by swadeley**
Hello

this blocks https://github.com/SatelliteQE/robottelo/pull/8012

test results will be there

### CODE REVIEW

**Review by mirekdlugosz — APPROVED** [review]
ACK

**Review by jameerpathan111 — APPROVED** [review]
ACK. need one more tier reviewer's ACK here.

File: airgun/entities/syncplan.py [hunk 1]
```diff
@@ -57,6 +57,18 @@ def add_product(self, entity_name, products_list):
...
 
+    def remove_product(self, entity_name, products_list):
+        """Remove product from sync plan
+
+        :param str entity_name: sync plan name
+        :param products_list: either one or list of products
+        """
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.products.resources.remove(products_list)
+        assert view.flash.is_displayed
```
Comment (omkarkhatavkar) [inline airgun/entities/syncplan.py:68]: This is the first time we are seeing the assert statement used here in Airgun PR Review.  @swadeley is your intention is that you wanted to see that error message?

**Review by omkarkhatavkar — APPROVED** [review]
ACK, one comment pending.

File: airgun/entities/syncplan.py (hunk 1 above)
Comment (swadeley) [reply airgun/entities/syncplan.py:68]: Hello @omkarkhatavkar 

I just wanted to make the new function consistent with the one above it: `def add_product`

Comment (mirekdlugosz) [reply airgun/entities/syncplan.py:68]: In current master, there are well over 200 `assert`s in Airgun module code:
```
$ ack -c assert airgun/ | cut -d ':' -f 2 | awk '{sum+=$1}END{print sum}'
216
```

So, `assert` is certainly not something exceptional.

**Comment by swadeley** [comment]
Thank you

**Pull request merged**